CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0
+
+The is a major release
+
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
```

Comment:
We're talking about different things.  I was talking about the version of *this* project which you bumped to 2.0.0.